projects/rocprofiler-sdk/tests/bin/openmp/CMakeLists.txt

```diff
@@ -23,11 +23,8 @@ project(rocprofiler-sdk-tests-bin-openmp LANGUAGES CXX)
 
 find_package(rocprofiler-sdk REQUIRED)
 
-set(DEFAULT_GPU_TARGETS "gfx906" "gfx908" "gfx90a" "gfx942" "gfx950" "gfx1100" "gfx1101"
-                        "gfx1102")
-
 set(OPENMP_GPU_TARGETS
-    "${DEFAULT_GPU_TARGETS}"
+    "${ROCPROFILER_GPU_TARGETS}"
```

Comment:
This is invalid when tests are build against an install of rocprofiler-sdk -- ROCPROFILER_GPU_TARGETS is not set. Furthermore, not all these targets may be supported by OpenMP.